mLRS/Common/hal/hal.h

```diff
@@ -39,8 +39,8 @@ following for the tx-hal and rx-hal files.
 In tx-hal files:
 
 #define DEVICE_HAS_DIVERSITY        // board supports diversity
-#define DEVICE_HAS_JRPIN5           // board has a pin for JR bay Pin5/SPort
-#define DEVICE_HAS_JRPIN5_FULL_DUPLEX   // board has full duplex uart for JR bay Pin5 function
+#define DEVICE_HAS_JRPIN5           // board has a pin for JR bay Pin5/SPort with tc interrupt
+#define DEVICE_HAS_JRPIN5_NO_TC     // board has half or full duplex uart for JR bay Pin5 function but no tc interrupt
```

Comment:
I understand your intention, but I find it cryptic
as far as I can see, this is pretty specific for ESP32 ... at least for the moment
so one could just use the old DEVICE_HAS_JRPIN5 and distunguish by ESP in the jrpin5 lib

I must say I also struggle a bit to understand how/why NO_TC can cover both full duplex and single wire, I would have thought there must be some pin managment which is not need for full duplex. But that's certainly my limitation LOL.

so, I would remove an use the old define and spilt out by esp32